js/views/modals/listingDetail/Listing.js

```diff
@@ -158,8 +158,37 @@ export default class extends BaseModal {
     const photoHash = photoCol[photoIndex].original;
     const phSrc = app.getServerUrl(`ipfs/${photoHash}`);
 
-    this.$photoSection.cropit('imageSrc', phSrc);
-    this.$photoSelected.attr('data-index', photoIndex);
+    this.$photoSelected.trigger('zoom.destroy'); // old zoom must be removed
+
+    this.$photoSelectedInner
+        .one('load', this.activateZoom(photoIndex, phSrc))
+        .attr('src', phSrc);
+  }
+
+  activateZoom(photoIndex, phSrc) {
+    setTimeout(() => {
+      // without a timeout the image hasn't rendered yet and has no height or width
+
+      if (this.$photoSelectedInner.width() >= this.$photoSelectedWidth ||
+          this.$photoSelectedInner.height() >= this.$photoSelectedHeight) {
+
+        this.$photoSelected
+            .attr('data-index', photoIndex)
+            .removeClass('unzoomable')
+            .zoom({
+              url: phSrc,
+              on: 'click',
+              onZoomIn: () => {
+                this.$photoSelected.addClass('open');
+              },
+              onZoomOut: () => {
+                this.$photoSelected.removeClass('open');
+              },
+            });
+      } else {
+        this.$photoSelected.addClass('unzoomable');
+      }
+    }, 500);
```

Comment:
It is available on load:

https://jsfiddle.net/p3z68ccv/2/

If it's not for you, it's because something else is happening. Have you checked the things I mentioned above to check?